rules/windows/credential_access_veeam_backup_dll_imageload.toml

```diff
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2024/03/14"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/03/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies potential credential decrypt operations by PowerShell or unsigned processes using the Veeam.Backup.Common.dll
+library. Attackers can use Veeam Credentials to target backups as part of destructive operations such as Ransomware
+attacks.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Veeam Credential Decrypt"
+risk_score = 47
+rule_id = "aaab30ec-b004-4191-95e1-4a14387ef6a6"
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Credential Access", "Data Source: Elastic Defend"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+library where host.os.type == "windows" and event.action == "load" and
+  (dll.name : "Veeam.Backup.Common.dll" or dll.pe.original_file_name : "Veeam.Backup.Common.dll") and
+  (
+    process.code_signature.trusted == false or
+    process.code_signature.exists == false or
+    process.name : "powershell.exe"
```

Comment:
Want to include scope to powershell_ise as well?

Reply:
Added